### DISCUSSION TITLE
Check layout space availability

### DISCUSSION THREAD

**Discussion by occoder** [h2oai/wave#678] [Q&A]
I'd like to fill out and remove the stat card from the page dynamically.
For sure, I could record the layout update history by my own application.
But it would be nicer if wave could provide API function that could help check what next available layout grid is or if certain grid is already taken.

**Comment by lo5** [comment]
@occoder - what you're trying to do might be better served by the flexible/responsive layout system introduced in 0.10, but I would suggest holding off until the 0.11 release (will have bugfixes and docs).

There are some work in progress examples here: https://github.com/h2oai/wave/tree/master/py/demo
